cmd/kubeadm/app/util/apiclient/idempotency.go

```diff
@@ -60,13 +60,30 @@ func CreateOrUpdateConfigMap(client clientset.Interface, cm *v1.ConfigMap) error
 // to conflicts, and a retry will be issued if the ConfigMap was modified on the server between the refresh and the update (while the mutation was
 // taking place)
 func CreateOrMutateConfigMap(client clientset.Interface, cm *v1.ConfigMap, mutator ConfigMapMutator) error {
-	if _, err := client.CoreV1().ConfigMaps(cm.ObjectMeta.Namespace).Create(cm); err != nil {
-		if !apierrors.IsAlreadyExists(err) {
-			return errors.Wrap(err, "unable to create ConfigMap")
+	var lastError error
+	err := wait.ExponentialBackoff(wait.Backoff{
+		Steps:    20,
+		Duration: 500 * time.Millisecond,
+		Factor:   1.0,
+		Jitter:   0.1,
```

Comment:
at some point we need to figure out how to make all api calls have some sort of uniformity in terms of backoff.

potentially:
```
func FuncionName(.... backoff *wait.Backoff)
```

where if `backoff` is `nil` a default backoff is used, a backoff that is the same for all api calls.

/lgtm

Reply:
Yes, my priority was to first make control plane join stable. After this PR I was thinking in sitting in front of all the requests we make and get to a proposal on how to make all this uniform and intuitive.